packages/fxa-settings/src/components/App/index.tsx

```diff
@@ -106,6 +108,22 @@ export const App = ({
     integration,
   ]);
 
+  if (integration && integration.isSync()) {
+    // send up fxaStatus request here
+    // merge sync account if config.sendFxAStatusOnSettings
+
+    if (isSyncDesktopOAuthIntegration(integration)) {
```

Comment:
See https://github.com/mozilla/fxa/pull/15723#discussion_r1438441593, this approach is still up for discussion.

Reply:
Thank you so much @LZoog , this looks good to me and sounds acceptable - we can configure the fxa_status on desktop to return the oauth query parameters and do that on every fxa_status where desktop is not yet signed in.

If this branch is good to build locally, I can have my local setup (which has Firefox Desktop supporting oauth) use this branch and verify if it works! Would also be good to do that now to uncover any unexpected issues